examples/local/README.md

```diff
@@ -21,7 +21,7 @@ guide, all placeholders must be replaced with sensible values.
 
 - *CLUSTER_NAME* - Cluster's name.
 - *COMMON_DOMAIN* - Cluster's etcd and API common domain.
-- *COMMON_DOMAIN_INGRESS* - Ingress common domain.
+- *COMMON_DOMAIN_INTERNAL* - Ingress common domain.
```

Comment:
This is great, thanks for correcting but `COMMON_DOMAIN_INTERNAL` is no longer needed.

This is because in early installs we had etcd and apiserver in `*.aws.giantswarm.io` and ingress in `*.aws.gigantic.io`. Now we use `gigantic.io` everywhere so only `COMMON_DOMAIN` is needed.

Reply:
Removed all instances of `COMMON_DOMAIN_INTERNAL`, also done in Wiki